hack/gen_protoc.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+. $ROOT_DIR/hack/utils.sh
+
+PROTO_PATH=${1}; shift
+PROTO_SRC=$@
+
+arch=$(get_arch)
+
+# if protoc-c is 1.4.x
+if protoc-c --version | grep -q -E "protobuf-c 1\.4\.[0-9]"; then
+    protoc-c --proto_path=$PROTO_PATH --c_out=. $PROTO_SRC
+else
+    echo "Generate proto files in docker"
+    docker run --rm \
+        --privileged=true \
+        -v $ROOT_DIR:/kmesh \
+        --name kmesh-build \
+        ghcr.io/kmesh-net/kmesh-build-${arch}:latest \
```

Comment:
Does the build image already support multi arch? @lec-bit

Reply:
https://github.com/orgs/kmesh-net/packages/container/package/kmesh-build

We have supported both arm and x86， so can remove arch suffix

Reply:
> https://github.com/orgs/kmesh-net/packages/container/package/kmesh-build
> 
> We have supported both arm and x86， so can remove arch suffix

then we should also remove the arch preffix in make build?